wordpress/mysql/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# =============================================================================
+# mysql:5.7
+#
+# Modification: WordPress MySQL error log Fail2Ban setup
+#
+# =============================================================================
+FROM mysql:5.7
+LABEL maintainer="Jacob Thompson <jacob.thompson@wiwasolvet.ca>"
+
+RUN apt-get update &&  \
+    apt-get install -y \
+    nano \
+    vim
+
+# Setup mysql auth log for Fail2Ban regex
+USER root
+RUN chown mysql:root /var/log/mysql/
+
+#RUN chmod -R 777 /var/log/mysql/
+
+RUN touch /var/log/mysql/error.log && \
+        printf "# This is the MySQL login authorization log." >> /var/log/mysql/error.log
+
+# Note: must create log folder and error.log file on first run, otherwise container will restart continually.
+
+RUN printf "%s\n" "[mysqld]" \
+        "log_warnings = 2" \
+        "log_timestamps = SYSTEM" \
+        "log_error = /var/log/mysql/error.log" >> /etc/mysql/my.cnf
+
+# service mysql restart
+
+# Sets timezone which normally uses dpkg-reconfigure tzdata (2 for America, 65 for Halifax)
+ENV TZ=America/Halifax
```

Comment:
Pass this via .env argument to docker-compose via ARG, to make regional association independent.